### ISSUE TITLE
WSL2跑jittor.test.test_resnet失败

### ISSUE THREAD

**Issue by Meidozuki** [Jittor/jittor#254]
能跑test_example但是test_resnet就被skipped

$ python3 -m jittor.test.test_example
[i 0726 19:53:34.030174 52 compiler.py:869] Jittor(1.2.3.79) src: /home/administrator/.local/lib/python3.8/site-packages/jittor
[i 0726 19:53:34.033356 52 compiler.py:870] g++ at /usr/bin/g++(9.3.0)
[i 0726 19:53:34.033490 52 compiler.py:871] cache_path: /home/administrator/.cache/jittor/default/g++
[i 0726 19:53:34.133293 52 __init__.py:286] Found addr2line(2.34) at /usr/bin/addr2line.
[i 0726 19:53:34.147420 52 compiler.py:958] py_include: -I/usr/include/python3.8 -I/usr/include/python3.8
[i 0726 19:53:34.159144 52 compiler.py:960] extension_suffix: .cpython-38-x86_64-linux-gnu.so
[i 0726 19:53:34.350068 52 compiler.py:1098] OS type:ubuntu OS key:ubuntu
[i 0726 19:53:34.350967 52 __init__.py:178] Total mem: 12.44GB, using 4 procs for compiling.
[i 0726 19:53:34.585605 52 jit_compiler.cc:22] Load cc_path: /usr/bin/g++
[i 0726 19:53:34.798999 52 compile_extern.py:444] mpicc not found, distribution disabled.
step 0, loss = 0.8328441381454468 {'hold_vars': 14, 'lived_vars': 64, 'lived_ops': 57}
step 1, loss = 0.5198176503181458 {'hold_vars': 14, 'lived_vars': 64, 'lived_ops': 57}
……
step 999, loss = 0.0009948197985067964 {'hold_vars': 14, 'lived_vars': 64, 'lived_ops': 57}
.
----------------------------------------------------------------------
Ran 1 test in 0.346s

OK

$ python3 -m jittor.test.test_resnet
[i 0726 19:54:06.905609 60 compiler.py:869] Jittor(1.2.3.79) src: /home/administrator/.local/lib/python3.8/site-packages/jittor
[i 0726 19:54:06.908316 60 compiler.py:870] g++ at /usr/bin/g++(9.3.0)
[i 0726 19:54:06.908415 60 compiler.py:871] cache_path: /home/administrator/.cache/jittor/default/g++
[i 0726 19:54:07.004902 60 __init__.py:286] Found addr2line(2.34) at /usr/bin/addr2line.
[i 0726 19:54:07.016443 60 compiler.py:958] py_include: -I/usr/include/python3.8 -I/usr/include/python3.8
[i 0726 19:54:07.028571 60 compiler.py:960] extension_suffix: .cpython-38-x86_64-linux-gnu.so
[i 0726 19:54:07.140669 60 compiler.py:1098] OS type:ubuntu OS key:ubuntu
[i 0726 19:54:07.141276 60 __init__.py:178] Total mem: 12.44GB, using 4 procs for compiling.
[i 0726 19:54:07.276948 60 jit_compiler.cc:22] Load cc_path: /usr/bin/g++
[i 0726 19:54:07.366589 60 compile_extern.py:444] mpicc not found, distribution disabled.
s
----------------------------------------------------------------------
Ran 1 test in 0.253s

OK (skipped=1)

**Comment by Jittor** [comment]
test_resnet是在有cuda的情况下才会跑，因为cpu训练略慢，您也可以主动删除其中跳过的代码`@unittest.skipIf(not jt.has_cuda, "Cuda not found")`, 强制在cpu上运行

**Comment by Meidozuki** [comment]
啊原来如此，谢谢。跟着快速开始，突然不能跑还以为哪里配置错了
